README.md

```diff
@@ -1,13 +1,25 @@
-# Project Auth
+<h1>🔐🔐 Project Auth 🔐🔐</h1>
 
-Replace this readme with your own information about your project.
+Pair-programming project developed by Noemi Frischknecht and Estefanía Quevedo.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project's goal was about buiding and **API with authentication** in order to implement a registration and login flow interface. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We learned how to authenticate users using tokens and how to store passwords in databases in a secure way. We also created a Frontend with a form to register, sign in and be able to see restricted content in case the login was successful.
+
+We used MongodB and Mongoose to build our Backend Express API. This API has 3 endpoints: 
+
+<ul>
+  <li>A POST endpoint to create a new user</li>
+  <li>A POST endpoint for the existing users to log in</li>
+  <li>A GET restricted endpoint which is the home of the restricted site's content. It uses an authentication middleware, so it's only accessible with a valid access token.</li>
+</ul>
```

Comment:
Thank you for this summary. It makes it super clear what to look for in your code 👍🔅